.github/workflows/python-package.yml

```
@@ -31,7 +31,8 @@ jobs:
         python -m pip install .[arrays,test]
     - name: Test source code and docs
       run: |
-        pytest --cov . --cov-report xml
+        cd tests
+        python -m pytest --cov=uncertainties --cov-report=xml
```

This nicely fixes what I broke when I made sure that the installed package was tested instead of the code in the repo. However, it excludes tracking coverage of the test files. We can track them as well by using a second `--cov`. I also think it is convenient to add `--cov-report=term` so that the percentages can be seen in the GitHub Actions output.

```suggestion
        python -m pytest --cov=uncertainties --cov=. --cov-report=xml --cov-report=term
```